x-pack/docs/en/security/authentication/configuring-pki-realm.asciidoc

```diff
@@ -2,26 +2,37 @@
 [[configuring-pki-realm]]
 === Configuring a PKI realm
 
-You can configure {es} to use Public Key Infrastructure (PKI) certificates
-to authenticate users. This requires clients to present X.509 certificates.
+You can configure {es} to use Public Key Infrastructure (PKI) certificates to
+authenticate users. This requires clients connecting directly to {es} to
+present X.509 certificates. The certificates must first be accepted for
+authentication on the SSL/TLS layer on {es}, before they are optionally further
+validated by a PKI realm. Users might also connect to {kib}, where they present
```

Comment:
```suggestion
validated by a PKI realm. Users might also connect to {kib} and provide
```

Reply:
I went with `and present`, I believe present is more customary than provide in this case.